NewForce-Cohort-5/tabloidmvc-the-daily-debuggers
# Bg edit categories

# Description

When you navigate to Categories after logging in, you can now click the pencil next to a category and a small form will show up where you can edit the name of a category.

## Type of change


# Testing Instructions

1. Clone down the repository.
2. `git fetch -a`
3. `git checkout bg-EditCategories`
4. Navigate to the root directory for this repo (directory-to)/tabloidmvc-the-daily-debuggers
5. Start TabloidMVC.sln
6. Make sure your db is created with seed data. Run [01_TabloidMVC_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidmvc-the-daily-debuggers/blob/main/SQL/01_TabloidMVC_Create_DB.sql) and [02_TabloidMVC_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidmvc-the-daily-debuggers/blob/main/SQL/02_TabloidMVC_Seed_Data.sql) queries in Visual Studio.
7. Start the server.
8. Log in as admin [admin@example.com](mailto:admin@example.com) (if not already logged in as admin).
9. Click Categories at the top of the page.
10. Click the pencil next to a category.  A tiny form with just an edit name field will be visible.  Edit the name of a category to test it.

# Checklist:
